doc/nrf/libraries/bluetooth_services/mesh/gen_dtt_cli.rst

```diff
@@ -12,17 +12,60 @@ The Generic DTT Client remotely controls the transition time state of a :ref:`bt
 The client can be used to set up the server's default transition time behavior when changing states in other models on the same element.
 
 Extended models
-===============
+***************
 
 None.
 
 Persistent storage
-==================
+******************
 
 None.
 
+Shell commands
+**************
+
+The Bluetooth mesh shell subsystem provides set of commands to interract with the Generic DTT Client model instantiated on a device.
+
+To make these commands available, enable :kconfig:`CONFIG_BT_MESH_SHELL` and :kconfig:`CONFIG_BT_MESH_SHELL_DTT_CLI`.
+
+``mdl_dtt instance get-all``
```

Comment:
Remove highlighting and replace h4 with h3. Or are we considering using a definition list instead?

Reply:
Replaced with definition list.